tests/tests_suite_test.go

```diff
@@ -44,6 +44,10 @@ var (
 )
 
 func getDir() string {
+	// if inside Docker container, don't return nothin'
```

Comment:
@arschles makes a good point about `DOCKERIMAGE`. I guess testing for `.dockerinit` is really relying on an implementation detail of Docker that may change someday. 

Sorry for the short-sighted advice. :crying_cat_face: